docs/namespaces.md

```diff
@@ -0,0 +1,126 @@
+
+`remote.php/dav/files/f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/path/to/file.txt` -> `/eos/f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/path/to/file.txt`
+ocdav service:
+DAV_FILES_NAMESPACE=`/eos/`
+                                                                storage provider
+                                                                    driver: `eoshome`
+                                                                    id: `1284d238-aa92-42ce-bdc4-0b0000009158`
+                                                                    layout: `{{substr 0 1 .Id.OpaqueId}}`
+                                                                            `f`
+                                                                    eos namespace (in eos): `/eos/dockertest/reva/users`
+                                                                    mount point `/eos`
+                                                                    full internal path: namespace + layout + relative path
+                                                                                        `/eos/dockertest/reva/users` + `f` + `f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/path/to/file.txt`
+
+simple example:
+
+`remote.php/webdav/path/to/file.txt` -> `/home/path/to/file.txt`
+ocdav service:
+WEBDAV_NAMESPACE=`/home/`
+                                        storage provider
+                                            driver: `eoshome`
+                                            layout: `{{.Id.OpaqueId}}`
+                                                    `f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c`
+                                            eos namespace (in eos): `/eos/dockertest/reva/users`
+                                            mount point `/home`
+                                            full internal path: namespace + layout + relative path
+                                                                `/eos/dockertest/reva/users` + `f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c` + `path/to/file.txt`
+
+`remote.php/dav/files/f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/path/to/file.txt` -> `/eos/f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/path/to/file.txt`
+ocdav service:
+DAV_FILES_NAMESPACE=`/eos/`
+                                                                        storage provider
+                                                                            driver: `eos`
+                                                                            eos namespace (in eos): `/eos/dockertest/reva/users`
+                                                                            mount point `/eos`
+                                                                            full internal path: namespace + layout + relative path
+                                                                                                `/eos/dockertest/reva/users` + `f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c/path/to/file.txt`
+
+ishank
```

Comment:
let's remove this bit.

let's replace everything with only the config that we decided to use in https://github.com/owncloud/ocis/pull/484

maybe add another set of configs for the OC storage

Reply:
hm it is now even more complacated because `/dav/files/<userid>` uses `/home` when `<userid>` matches the currently logged in user...